pkg/config/appsync_server_config.go

```diff
@@ -41,7 +42,9 @@ type AppSyncConfig struct {
 	RateLimit uint64 `env:"RATE_LIMIT,default=60"`
 
 	// AppSync config
-	AppSyncURL string `env:"APP_SYNC_URL, default=https://www.gstatic.com/exposurenotifications/apps.json"`
```

Comment:
Should we consider a world where app syncs are actually a database table and it's possible to sync from multiple URLs?

Reply:
I think the single URL is fine.